### PR TITLE
Fix usecase_vpn_test tests are failing inconsistently

### PR DESCRIPTION
**Pull request by denis-tingajkin**
## Motivation and Context
https://github.com/networkservicemesh/networkservicemesh/issues/1152

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by nickolaev** [comment]
I think that increasing the timeouts shoudl fix it, like this:

https://github.com/networkservicemesh/networkservicemesh/pull/1269/commits/21740140690ee8da4eb75e9eff1d63c1beeaf975

File: test/integration/usecase_vpn_test.go [hunk 1]
```diff
@@ -125,8 +125,8 @@ func testVPN(t *testing.T, ptnum, nodesCount int, affinity map[string]int, verbo
 	}
 
 	pingCommand := "ping"
-	addressPool := "10.60.1.0/24"
-	srcIP, dstIP := "10.60.1.1", "10.60.1.2"
+	addressPool := "172.16.1.0/24"
+	srcIP, dstIP := "172.16.1.1", "172.16.1.2"
```
Comment (nickolaev) [inline test/integration/usecase_vpn_test.go]: @denis-tingajkin can you elaborate in the PR body why this is the fix?

Comment (haiodo) [reply test/integration/usecase_vpn_test.go]: Also could we check this dynamically for test to choose exclude prefixes for cluster we are running on?

Comment (haiodo) [reply test/integration/usecase_vpn_test.go]: Ok discussed, according to @lobkovilya we already have tests with same values, and in future we could think about moving to dynamic obtaining of this values.

**Pull request merged**